bigquery/tests/unit/test__helpers.py

```diff
@@ -468,15 +468,36 @@ def test_w_array_of_struct_w_array(self):
                 {u'first': [5, 6], u'second': 7},
             ],))
 
+    def test_row(self):
+        from google.cloud.bigquery._helpers import Row
+
+        VALUES = (1, 2, 3)
+        r = Row(VALUES, {'a': 0, 'b': 1, 'c': 2})
+        self.assertEqual(r.a, 1)
+        self.assertEqual(r[1], 2)
+        self.assertEqual(r['c'], 3)
+        self.assertEqual(len(r), 3)
+        self.assertEqual(r.values(), VALUES)
+        self.assertEqual(repr(r),
+                         "Row((1, 2, 3), {'a': 0, 'b': 1, 'c': 2})")
```

Comment:
This test is failing because the order of dictionary elements isn't guaranteed.

It might not be a bad idea to sort them alphabetically in `__repr__`. Or just pass in an [OrderedDict](https://docs.python.org/2/library/collections.html#collections.OrderedDict) in this test.

Reply:
I sorted them by value (index), since that will be the order they appear in the schema.